acceleration/distributed_training/unet_evaluation_workflows.py

```diff
@@ -22,11 +22,9 @@
   `--master_addr="192.168.1.1"`
   `--master_port=1234`
   For more details, refer to https://github.com/pytorch/pytorch/blob/master/torch/distributed/launch.py.
-  Alternatively, we can also use `torch.multiprocessing.spawn` to start program, but it that case, need to handle
```

Comment:
Thanks for the update.
Do you mean we can't use `torch.multiprocessing.spawn` anymore if using ignite?
CC @vfdev-5 .

Thanks.

Reply:
@Nic-Ma I think it is unrelated to ignite. If you would like to replace torch distributed launch by `torch.multiprocessing.spawn`, user needs to update the code before running it as `python unet_evaluation_workflows.py --arg-num-proc-per-node=NUM_GPUS_PER_NODE --arg-nnodes=NUM_NODES --arg-node_rank=INDEX_CURRENT_NODE --arg-master-addr --arg-master-port` and also translate that into world size and rank. Ignite does that for the user like that: 
- spawn and launch unifiied approch: https://pytorch.org/ignite/distributed.html#ignite.distributed.launcher.Parallel please see examples below
- only spawning: https://pytorch.org/ignite/distributed.html#ignite.distributed.utils.spawn

Reply:
@vfdev-5 , thanks for your explanation!
@YuanTingHsieh , so I think you should not delete the previous doc-string about the `spawn` method.

Thanks.

Reply:
A typo:
```diff
- but it that case
+ but in that case
```